modules/docs/mdx/6.0-MIGRATION-GUIDE.mdx

```diff
@@ -0,0 +1,111 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+<Meta title="Welcome/Migration Guides/v6.0" />
+
+# Canvas Kit 6.0 Migration Guide
+
+Below are the breaking changes made in Canvas Kit v6. Please
+[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
+any questions about the update.
+
+- [Codemod](#codemod)
+- [Component Deprecations](#component-deprecations)
+  - [PageHeader](#page-header)
+
+## Codemod
+
+Please use our [codemod package](https://github.com/Workday/canvas-kit/tree/master/modules/codemod)
+to automatically update your code to work with a majority of the breaking changes in the migration
+from Canvas Kit v5 to v6. Simply run:
+
+```sh
+> npx @workday/canvas-kit-codemod v6 [path]
+```
+
+> Note: This codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` extensions. You may need to make
+> some manual changes in other file types (`.json`, `.mdx`, `.md`, etc.).
+
+> Note: You may need to run your linter after executing the codemod, as it's resulting formatting
+> (spacing, quotes, etc.) may not match your project's styling.
+
+**Breaking changes accounted for by this codemod will be marked with a 🤖.**
+
+**Please verify all changes made by the codemod. As a safety precaution, we recommend committing the
+changes from the codemod as a single isolated commit (separate from other changes) so you can
+rollback more easily if necessary.**
+
+[Let us know](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you
+encounter any issues or use cases that we've missed. The `@workday/canvas-kit-codemod` package will
+help us maintain additional codemod transforms to make future migrations easier.
+
+## Component Deprecations
+
+### Deprecation Types
+
+There are two main deprecation types: soft and hard.
+
+#### Soft Deprecation
+
+During a soft deprecation, a component is still available with its full functionality, but it will
+be renamed with a prefix to indicate its soft-deprecated status. It will also include a console
+warning to announce the deprecation. This warning will only be triggered on the component's initial
+render. Soft-deprecated types or utilities that will also be renamed but will generally have no
+console warning.
```

Comment:
I wasn't sure if the last sentence about soft-deprecated types and utilities just had an extra "that" or if you meant to say "that [something] will also be renamed..." -- I just assumed it was an extra "that."

```suggestion
A soft-deprecated component is still available with its full functionality, but it will have been
renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
to announce its deprecation. This warning will only be triggered on the component's initial render.

Soft-deprecated types and utilities will also be renamed but generally will not trigger a console
warning.
```